src/components/CalendarModal/CalendarModal.module.scss

```diff
@@ -6,22 +6,66 @@
   height: 100%;
   background: rgba(0, 0, 0, 0.6);
   text-align: center;
-  z-index: 1;
+  z-index: 10;
 }
 
 .modal-main {
   padding: 1rem;
   position: fixed;
   background: white;
-  width: 80%;
+  max-width: 500px;
+  width: 60%;
   height: auto;
   top: 50%;
   left: 50%;
   transform: translate(-50%, -50%);
+  border-radius: 10px;
+}
+
+.modal-main button {
+  background-color: white;
 }
 
 .options {
   display: flex;
+  flex-direction: column;
   gap: 1rem;
   justify-content: center;
+  position: center;
```

Comment:
If we add `align-items: center` to this flexbox (`.options`), all children will be center aligned horizontally so we can avoid any hacky solutions to center the content.

position: center can also be deleted